packages/lesswrong/components/linkPreview/PostLinkPreview.tsx

```diff
@@ -666,12 +666,65 @@ const MetaculusPreviewComponent = registerComponent('MetaculusPreview', Metaculu
   styles: metaculusStyles
 })
 
+const fatebookStyles = (theme: ThemeType): JssStyles => ({
```

Comment:
remove `: JssStyles` return type annotation and add `<typeof fatebookStyles` to the `classes` prop in the component